index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Document</title>
+</head>
+<body>
+  <h1>Hello Microverse!</h1>
+</body>
```

Comment:
You seem to have missed the <header> tags that should wrap the h1 content, which is a requirement for the project.

[    - A .html file that includes a header with the text "Hello Microverse!". Use `h1` tag for that.](https://github.com/microverseinc/curriculum-transversal-skills/blob/5cb3d2611298d05ddb5eaf2abf41524555639a76/documentation/hello_microverse_project.md?plain=1#L42)